include/cute/arch/mma_sm90_gmma.hpp

```diff
@@ -84,15 +86,15 @@ warpgroup_fence_operand(uint32_t& reg) {
   // MSVC emits a build error for 'asm volatile'
   // even if it only occurs in a __device__ function.
   // This prevents the error.
-#if defined(__CUDA_ARCH__)
+#if defined(__CUDA_ARCH__) || defined(__SYCL_CUDA_ARCH__)
```

Comment:
This __SYCL_CUDA_ARCH__ seems to create  a lot of noise in the code can we we wrap it up with __cuda_arch__